yarn.lock

```diff
@@ -2042,13 +2049,6 @@ passport-facebook@^2.0.0:
   dependencies:
     passport-oauth2 "1.x.x"
 
-passport-local@^1.0.0:
-  version "1.0.0"
```

Comment:
Were there any changes between passport-local v1 to v2 to account for?

Reply:
I just realized that we actually bumped the version twice in our fork: https://github.com/artsy/passport-local-with-otp

Reply:
Passport-local never made it past 1.0.0, [for better or worse](https://artsyproduct.atlassian.net/browse/TRUST-215?filter=-2)

The version bumps in our fork were for local releases as I was `yalc`-publishing to test things out. Since it's all <1 I plan to just leave as is.